testscripts/lockfile/nopaths.txt

```diff
@@ -0,0 +1,23 @@
+# Test installing a package without outputs in the store path. 
+# NOTE: Purposefully using a weird version to ensure it is not already in store.
```

Comment:
once I run this testscript locally, it will then be in my store indefinitely (or until I run `nix store gc`) So, in that case the testscript won't be testing that its missing from local nix store.

so, the goal of this testscript is then to verify that one can do `devbox install` (equivalent) of "a package that doesn't have ooutputs in store path" ? Feels reasonable.

Reply:
yeah, this test mostly just tests that devbox install/run/shell work if the lockfile is missing store paths. I think it is useful (even though it would not catch this particular case where this function was removing store paths that were not installed) 

The unit test you had originally (with valid: false) does catch it! I just misunderstood the API, and broke your test.